publish-report/action.yml

```diff
@@ -10,33 +10,41 @@ inputs:
     required: false
     description: "Path to the folder to publish. Defaults to 'playwright-report/'"
 
-
 runs:
   using: "composite"
   steps:
     - id: get-secrets
-      uses: grafana/shared-workflows/actions/get-vault-secrets@main
+      uses: grafana/shared-workflows/actions/get-vault-secrets@28361cdb22223e5f1e34358c86c20908e7248760 # get-vault-secrets-v1.1.0
```

Comment:
maybe this should keep using the latest version available?

Reply:
Nooo... published 18hrs ago. 🫨 

I'll update it.

Reply:
Tis updated in https://github.com/grafana/plugin-actions/pull/62/commits/70a1f4f3ecb70f8f2e5940994aee7d0dd956f802 👍